custom_modules/application/window.js

```diff
@@ -1,15 +1,15 @@
 class Window {
   constructor() {
-    this.ItemList = document.querySelector('.meal-content');
+    this.itemList = document.querySelector('.meal-content');
```

Comment:
- [x] The code follows JavaScript Best practices

Reply:
Thanks 😄